fec.config.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
       './useArhChatbot': path.resolve(__dirname, './src/aiClients/useArhClient.ts'),
       './useRhelChatbot': path.resolve(__dirname, './src/aiClients/useRhelLightSpeedManager.ts'),
       './useVaChatbot': path.resolve(__dirname, './src/aiClients/useVaManager.ts'),
+      './VirtualAssistantStateSingleton': path.resolve(__dirname, './src/utils/VirtualAssistantStateSingleton.ts'),
```

Comment:
I think we should start making a bit more sense with the "import paths". Can we do somethin like "./state/globalState" for the module? When we start leaning more into these tiny modules it will quickly become hard to remember everything.